oauth2client/contrib/appengine.py

```diff
@@ -89,11 +89,11 @@ def _safe_html(s):
     return cgi.escape(s, quote=1).replace("'", '&#39;')
 
 
-class InvalidClientSecretsError(Exception):
-    """The client_secrets.json file is malformed or missing required fields."""
+class Error(Exception):
+    """Base error for this module."""
 
 
-class InvalidXsrfTokenError(Exception):
+class InvalidXsrfTokenError(Error):
```

Comment:
So from what I can tell, this is only used in this class. It's raised in the OAuth2 callback which is a request handler, so the user of this library never really gets a chance to catch it - it likely bubbles up as a top-level exception.

I'd recommend we excise it and deal with the bad token the same way as flask util: 
https://github.com/google/oauth2client/blob/master/oauth2client/contrib/flask_util.py#L410

(As an aside, can you file a bug to split the webapp2 parts of this into its own module?)